### ISSUE TITLE
Failed to migrate from the RC version to the 1.1.0

### ISSUE THREAD

**Issue by glvecino** [IdentityServer/Thinktecture.IdentityServer3#900]
Hello,
We have a ASP.NET MVC + WebApi application, developed with the IdentityServer Beta2, and migrated to the RC version, and it's working fine.
Now we tried to migrate from RC to 1.1.0 but we are getting an error:
"An error occurred when trying to create a controller of type 'WelcomeController'. Make sure that the controller has a parameterless public constructor."
This happens when we add the Core project (1.1.0 from GitHub) as a reference in our project (just like we have done with Beta2 and RC).
If we instead of adding the Core project as a reference, we add a reference to the library IdentityServer3.dll from the Nuget Package IdentityServer3 1.1.0 all works fine.

Any ideas of what can be happening?

Thanks

**Comment by brockallen** [comment]
Not sure -- usually the exception from the SI system indicates the param that it could not map. 

Also, it's best if you reference IdSvr via NuGet and not via source code, so if you take that approach then it's much easier for us in helping you.

**Comment by gilm0079** [comment]
I saw this was closed already.  I'm seeing the same error from DI on the welcome controller.  Same situation. Nuget works, source doesn't.  Any ideas?  I would usually use the Nuget, but trying to get the source working so I can troubleshoot why my app is getting an invalid_request response...

**Comment by leastprivilege** [comment]
I would use the core source project if you need to debug the source. As for troubleshooting - check the logs.

**Comment by gilm0079** [comment]
I think I found the cause of this issue with using the source in case anyone else has the same error.  It is a VS issue.  The core project file doesn't have a default start URL set so I'm guessing VS is trying to guess at which file it should load in the browser.  I changed the startup page to be "core" so it starts the browser at the /core and that seems to work fine.

Dominick - maybe it would be worth adding a default root path map to the core map path via the startup.cs file like you guys have in the IdentityManager project.

// used to redirect to the main admin page visiting the root of the host
            app.Run(ctx =>
            {
                ctx.Response.Redirect("/core/");
                return System.Threading.Tasks.Task.FromResult(0);
            });
